src/esportsbot/db_gateway.py

```diff
@@ -63,6 +64,7 @@ def insert(self, table, params):
             db.close()
             return True
         except Exception as err:
+            traceback.print_exc()
```

Comment:
Is there some way we can standardise error handling like this?

Reply:
I've now switched all of my traceback printing to use `lib.exceptions.print_exception_trace`